app/handlers/sessions_callback.rb

```diff
@@ -76,6 +76,7 @@ def handle
     end
 
     outputs[:status] = status
+    outputs[:authentication_provider] = authentication.provider
```

Comment:
could just store the authentication and call `.provider` later (in case it becomes useful for someone else to have it.  obviously not big deal tho